lib/src/soloud.dart

```diff
@@ -2578,7 +2579,8 @@ interface class SoLoud {
       return;
     }
 
-    if (!_log.isLoggable(Level.SEVERE)) {
+    if (!_log.isLoggable(Level.SEVERE) &&
+        playerError != PlayerErrors.maxActiveVoiceCountReached) {
       // Do not do extra work if the logger isn't listening.
       return;
     }
```

Comment:
```suggestion
    // Do not do extra work if the logger isn't listening
    // to the appropriate level.
    final logLevel = playerError == PlayerErrors.maxActiveVoiceCountReached
        ? Level.INFO
        : Level.SEVERE;

    if (!_log.isLoggable(logLevel)) {
      return;
    }
```

I got lost in this logic so I'm suggesting this change. It goes with the suggestion below.